basicHTML- v.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" name="viewport" content="width=device-width, initial-scale=1.0">
+		<title> Multimedia </title>
+	</head>
+	
+	<body>
+		<center>
+		<div align="center" style="padding:20px">
+		<h1> WELLCOME </h1>
+        
+        <img src="http://www.larotativadigital.com.ar/wp-content/uploads/2017/01/globant_logo.jpg" class="responsive" alt="globant_logo">
+
+        <audio src="hype_home.mp3" controls alt="music"></audio> </br>
+        
+        <video src="https://www.youtube.com/watch?v=4JkIs37a2JE" controls poster="sintel.jpg" width="600" alt="musical video"></video>
+		</div>
```

Comment:
Take a look at code indentations. It is important to organize your code when you work in a team to make the code more understandable.